5b1a7e10-bc41-4dfd-9038-9b9eb589d78d.json

```diff
@@ -1,14 +1,73 @@
-﻿{
+{
   "UUID": "5b1a7e10-bc41-4dfd-9038-9b9eb589d78d",
   "Categoria": "",
+  "Categorie Google": [
+    "point_of_interest",
+    "establishment"
+  ],
   "Ragione Sociale": "CAMERA DI COMMERCIO INDUSTRIA AGRI COLTURA E AGRICOLTURA",
-  "Indirizzo": "VIA MATTEOTTI N.32 Int.1",
-  "Telefono": "0362/48031",
+  "Nome Visualizzato": "Camera di Commercio di Monza e Brianza",
+  "Indirizzo": "Piazza Cambiaghi, 9, 20900 Monza MB, Italy",

```

Comment:
@ipersoft l'indirizzo trovato è di Monza e non trovo riferimenti certi dell'attività su Internet. Chiudo la PR anche perché possiamo aggiungere le attività mancanti in seguito.